src/Input/Select/Select.tsx

```diff
@@ -157,7 +157,7 @@ const Select: React.FC<Props> & {
         setIsFocus(false);
       }
     };
-    document.addEventListener('click', onClickOutside, false);
+    document.addEventListener('mousedown', onClickOutside, false);
     return () => document.removeEventListener('click', onClickOutside, false);
```

Comment:
this line needed to be change to `mousedown` as well. 

This fix is so smart! Thank u @westwood846

Reply:
Thanks :)